tests/test_loadReferenceObjects.py

```diff
@@ -26,30 +26,22 @@
 
 import lsst.afw.table as afwTable
 import lsst.log
-from lsst.meas.algorithms import ReferenceObjectLoaderBase, getRefFluxField, getRefFluxKeys
+from lsst.meas.algorithms import ReferenceObjectLoader, getRefFluxField, getRefFluxKeys
 from lsst.meas.algorithms.loadReferenceObjects import hasNanojanskyFluxUnits, convertToNanojansky
 import lsst.pex.config
 import lsst.utils.tests
 
 from ingestIndexTestBase import makeConvertConfig
 
 
-class TrivialLoader(ReferenceObjectLoaderBase):
-    """Minimal subclass of ReferenceObjectLoaderBase to allow instantiation
-    """
-
-    def loadSkyCircle(self, ctrCoord, radius, filterName):
-        pass
-
-
-class ReferenceObjectLoaderBaseTestCase(lsst.utils.tests.TestCase):
-    """Test case for ReferenceObjectLoaderBase abstract base class.
+class ReferenceObjectLoaderTestCase(lsst.utils.tests.TestCase):
+    """Test case for ReferenceObjectLoader.
```

Comment:
This line isn't necessary: that's obvious from the name. The sentence below probably could go too, since there's no more ABC. Maybe rename or put in a sentence about how this tests the more generic parts of it, not the actual catalog loading.